tutoraspects/templates/openedx-assets/assets/dashboards/Instructor_dashboard.yaml

```diff
@@ -231,12 +234,12 @@ metadata:
     filterType: filter_select
     id: NATIVE_FILTER-V77Ybaw2w
     name: Video name
-    requiredFirst: true
     scope:
       excluded: []
       rootPath:
       - TAB-CLiLC4zxo
-    tabsInScope: []
+    tabsInScope:
+    - TAB-CLiLC4zxo
```

Comment:
Not sure, but same comment here

Reply:
In this case I think it's intended. The video name filter only applies to "video drilldown" tab.